custom/icds_reports/static/js/icds_dashboard_utils.js

```diff
@@ -0,0 +1,33 @@
+/* global angular hqImport */
+
+window.populateDashboardConstants = function populateDashboardConstants() {
```

Comment:
I'm pretty sure this should be using hqDefine, though I was largely following the pattern used by the rest of the dashboard...

Reply:
Yeah, the more modular way to do this is to wrap this whole file like
```
hqDefine("js/icds_dashboard_utils", function () {
    ...
    return {
        populateDashboardConstants: populateDashboardConstants,
    };
});
```

And then call it with `hqImport("js/icds_dashboard_utils").populateDashboardConstants()`

Reply:
thanks, I'll try that.

does compression work properly with hqImport blocks?

Reply:
yeah, compression should be fine